submissions/Liddell777/hooli-style-popup/index.html

```diff
@@ -0,0 +1,118 @@
+                                </a>
+                                <a class="mainpopup__link" href="#">
+                                    <img src="./img/menupopup/app.png" alt="apps icon">
+                                    Apps
+                                </a>
+                                <a class="mainpopup__link" href="#">
+                                    <img src="./img/menupopup/docs.png" alt="docs icon">
+                                    Document
+                                </a>
+                                <input type="checkbox" id="mainpopup__input" class="mainpopup__item-input" />
+                                <label for="mainpopup__input" class="mainpopup__item-label" tabindex="0"></label>
+                                <a class="mainpopup__link mainpopup__link--hidden" href="#">
+                                    <img src="./img/menupopup/menumore/cloud-network.png" alt="cloud network">
+                                    Cloud
+                                </a>
+                                <a class="mainpopup__link mainpopup__link--hidden" href="#">
+                                    <img src="./img/menupopup/menumore/gear.png"  alt="gear">
+                                    Gear
+                                </a>
+                                <a class="mainpopup__link mainpopup__link--hidden" href="#">
+                                    <img src="./img/menupopup/menumore/group.png"  alt="group">
+                                    Group
+                                </a>
+                                <a class="mainpopup__link mainpopup__link--hidden" href="#">
+                                    <img src="./img/menupopup/menumore/music-folder.png" alt="music folder">
+                                    Music
+                                </a>
+                                <a class="mainpopup__link mainpopup__link--hidden" href="#">
+                                    <img src="./img/menupopup/menumore/shield.png"  alt="shield">
+                                    Shield
+                                </a>
+                            </div>
+                        </div>
+                    </li>
+                    <li class="menu__item">
+                        <a class="menu__link-bell" href="#"><img src="img/menu/bell.png" alt="bell"></a>
+                    </li>
+                    <li class="menu__item">
+                        <a class="menu__link-user" href="https://github.com/Liddell777"><img src="img/menu/45641044.jpg" alt="picture"></a>
+                    </li>
+                </ul>
+            </nav>
+        </header>
+        <main>
+            <div class="container">
+                <div class="search__panel">
+                    <input class="container__input" type="search" placeholder="Feed me a leather sack">
+                    <input type="submit" class="container__btn" value="">
```

Comment:
Wrap both these input in HTML tag `form`.
`input type="submit"` is ok in case when you wrapped your inputs by tag form

Reply:
Hi!
I added to mine input teg form and i hope in this version its correct
form class="search__panel" action="#" method="get"
input class="container__input" type="search" placeholder="Feed me a leather sack">
input type="submit" class="container__btn" value=""
/form